jupyterhub/requirements-k8s.txt

```diff
@@ -16,6 +16,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+-e git://github.com/jupyterhub/kubespawner.git@46a4b109c5e657a4c3d5bfa8ea4731ec6564ea13#egg=jupyterhub-kubespawner
```

Comment:
can you add a comment here that installing from git is temporary until a new version of kubespawner is released?

Reply:
Done